lib/ryzenadj.h

```diff
@@ -60,6 +65,13 @@ EXP void CALL cleanup_ryzenadj(ryzen_access ry);
 
 EXP enum ryzen_family get_cpu_family(ryzen_access ry);
 EXP int get_bios_if_ver(ryzen_access ry);
+EXP uint32_t get_table_ver(ryzen_access ry);
+EXP size_t get_table_size(ryzen_access ry);
+//get table addr can be used to check for ptable support, because if you get a dram address, all preconditions are met
+EXP uint32_t get_table_addr(ryzen_access ry);
+
+EXP int CALL refresh_pm_table(ryzen_access ry);
```

Comment:
@FlyGoat is this name ok?
I did try to select one which is meaningful for the api user.

Reply:
Well implementation details shouldn't be exported to users.
Unsupported ptable could be detected by return error when refreshing table.